{"uid":"c7fdd5e3213b9150","name":"Oops...we found an error preparing this test file: test.spec.js","fullName":"Oops...we found an error preparing this test file: test.spec.js","time":{"start":1656129590313,"stop":1656129590313,"duration":0},"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress/support/index.js\n\n","statusTrace":"\n\nError: Webpack Compilation Error\n./cypress/support/hooks.js\nModule build failed (from /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/babel-loader/lib/index.js):\nSyntaxError: /home/runner/work/NPM-Cypress/NPM-Cypress/cypress/support/hooks.js: Unexpected token (10:78)\n\n   8 |\n   9 | beforeEach(function () { \n> 10 |     if(this.currentTest.state = true && fs.readdirSync('/screenshots').length))\n     |                                                                               ^\n  11 |         cy.task('deleteFolder');\n  12 | });\n  13 |\n @ ./cypress/support/index.js 5:0-18\n    at handle (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/packages/server/node_modules/@cypress/webpack-preprocessor/dist/index.js:180:23)\n    at finalCallback (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:257:39)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:304:22\n    at Compiler.emitRecords (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:499:39)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:298:10\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:482:27\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/neo-async/async.js:2818:7\n    at done (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/neo-async/async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:464:33\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/graceful-fs/graceful-fs.js:143:16\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/graceful-fs/graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (fs.js:156:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress/support/index.js\n\n","statusTrace":"\n\nError: Webpack Compilation Error\n./cypress/support/hooks.js\nModule build failed (from /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/babel-loader/lib/index.js):\nSyntaxError: /home/runner/work/NPM-Cypress/NPM-Cypress/cypress/support/hooks.js: Unexpected token (10:78)\n\n   8 |\n   9 | beforeEach(function () { \n> 10 |     if(this.currentTest.state = true && fs.readdirSync('/screenshots').length))\n     |                                                                               ^\n  11 |         cy.task('deleteFolder');\n  12 | });\n  13 |\n @ ./cypress/support/index.js 5:0-18\n    at handle (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/packages/server/node_modules/@cypress/webpack-preprocessor/dist/index.js:180:23)\n    at finalCallback (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:257:39)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:306:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:304:22\n    at Compiler.emitRecords (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:499:39)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:298:10\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:485:14\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:482:27\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/neo-async/async.js:2818:7\n    at done (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/neo-async/async.js:3522:9)\n    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:4:1)\n    at AsyncSeriesHook.lazyCompileHook (/home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/tapable/lib/Hook.js:154:20)\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/webpack/lib/Compiler.js:464:33\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/graceful-fs/graceful-fs.js:143:16\n    at /home/runner/.cache/Cypress/9.7.0/Cypress/resources/app/node_modules/graceful-fs/graceful-fs.js:61:14\n    at FSReqCallback.oncomplete (fs.js:156:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","steps":[],"attachments":[{"uid":"79e2e58e1b94727a","name":"video recording","source":"79e2e58e1b94727a.mp4","type":"video/mp4","size":198698}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"suite","value":"test.spec.js"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"c7fdd5e3213b9150.json","parameterValues":[]}